### PR TITLE
New package: escrotum-0.2.1

### PR DESCRIPTION
**Pull request by NoctuaNivalis**

### CODE REVIEW

File: srcpkgs/escrotum/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'escrotum'
+pkgname=escrotum
+version=0.2.1
+revision=2
```
Comment (Vaelatern) [inline srcpkgs/escrotum/template]: Revision 1.

File: srcpkgs/escrotum/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+build_style=python-module
+python_versions="2.7"
+#conf_files=""
+hostmakedepends="python-setuptools"
+makedepends=""
+depends="python pygtk"
```
Comment (Vaelatern) [inline srcpkgs/escrotum/template]: `depends="pygtk"`

**Comment by Vaelatern** [comment]
Please fix xlint errors. https://travis-ci.org/voidlinux/void-packages/jobs/119187945

**Comment by NoctuaNivalis** [comment]
Fixed the remarks. I incremented the revision because I needed a second try to get this working (I forgot the pygtk dependency). I force-pushed this not to mess up the commit messages.

**Comment by xtraeme** [comment]
This pkg is missing `noarch=yes`

**Comment by NoctuaNivalis** [comment]
Fixed.

File: srcpkgs/escrotum/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'escrotum'
+pkgname=escrotum
+version=0.2.1
+revision=1
+noarch=yes
+build_style=python-module
+python_versions="2.7"
```
Comment (Vaelatern) [inline srcpkgs/escrotum/template]: Needs a line `pycompile_module="escrotum"`

Comment (Vaelatern) [inline srcpkgs/escrotum/template]: added just below this.

**Pull request merged**